amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/jscript/app/JSONConversionComponent.java

```diff
@@ -76,6 +79,7 @@
  *
  * @author Roy Wetherall
  */
+@Slf4j
```

Comment:
This is cool - I've not seen this annotation before.

Reply:
Thank you for noticing.